Dockerfile

```diff
@@ -286,7 +238,7 @@ ${CONFIG_OPTION} = {{ getv("${TEMPLATE_OPTION}") }}
 {% endif %}
 EOS
 done
-# Add clustering settings (keep logic in remco for DNS lookups of SERVICE_NAME)
+# Add clustering settings (keep template logic in remco for DNS lookups of SERVICE_NAME)
```

Comment:
Looked like in other parts you were removing remco. I know this is a comment though.